backend/src/event/EVENT_SUBSCRIBE_NEWSLETTER.ts

```diff
@@ -0,0 +1,7 @@
+import { Event as DbEvent } from '@entity/Event'
+import { User as DbUser } from '@entity/User'
+
+import { Event, EventType } from './Event'
+
+export const EVENT_SUBSCRIBE_NEWSLETTER = async (user: DbUser): Promise<DbEvent> =>
```

Comment:
I suggest to name the events so they are grouped:
- EVENT_NEWSLETTER_SUBSCRIBE
- EVENT_NEWSLETTER_UNSUBSCRIBE